stats/misc_stats1.php

```diff
@@ -5,11 +5,14 @@
 include_once($relPath.'theme.inc');
 include_once($relPath.'page_tally.inc'); // get_page_tally_names()
 include_once($relPath.'misc.inc'); // get_enumerated_param()
+include_once($relPath.'graph_data.inc');
 
 require_login();
 
 $valid_tally_names = array_keys(get_page_tally_names());
 $tally_name = get_enumerated_param($_GET, 'tally_name', null, $valid_tally_names);
+$start = get_param_matching_regex($_GET, 'start', null, '/$(\d{4}-\d{1,2})?$/', true);
```

Comment:
I think this regex should start with `^` to anchor it at the beginning.

Reply:
Since we're passing these in as-is to the SQL we want to be sure and test to get these right.